docs/how-to-guides/column-mapping.md

```diff
@@ -0,0 +1,64 @@
+# Column mapping
+
+In this document, we will introduce how to map inputs with column mapping when running a flow.
+
+## Column mapping introduction
+
+Column mapping is a mapping from flow input name to specified values.
+If specified, the flow will be executed with provided value for specified inputs.
+The following types of values in column mapping are supported:
+
+- `${data.<column_name>}` to reference from your test dataset.
+- `${run.outputs.<output_name>}` to reference from your flow output.
+- `STATIC_VALUE` to create static value for all lines for specified column.
+
+## Flow inputs override priority
+
+Flow input value are override based on the following priority:
+
+"specified in column mapping" > "default value" > "same name column in provided data".
+
+For example, if we have a flow with following inputs:
+
+```yaml
+inputs:
+  input1:
+    type: string
+    default: "default_val1"
+  input2:
+    type: string
+    default: "default_val2"
+  input3:
+    type: string
+  input4:
+    type: string
+...
+```
+
+With the following data
+
+```json
+{"input3": "val3_in_data", "input4": "val4_in_data"}
+```
+
+And use the following YAML to run
+
+```yaml
+$schema: https://azuremlschemas.azureedge.net/promptflow/latest/Run.schema.json
+flow: path/to/flow
+# my_flow has default value val2 for key2
+data: path/to/data
+# my_data has column key3 with value val3
+column_mapping:
+    input1: "val1_in_column_mapping"
+    input3: ${data.input3}
+```
+
+The run details will be like this:
```

Comment:
do the "pf run show-details **" command open to client ? do they need to understand this command and its output?

Reply:
Yep, it's introduced in here: https://microsoft.github.io/promptflow/how-to-guides/manage-runs.html#show-run-details